charts/platform-system/values.yaml

```diff
@@ -9,11 +9,26 @@ prefix: ""
 
 apps:
   istio:
-    enabled: true
+    enabled: false
     namespace: istio-system
     nodeSelector: {}
     tolerations: []
     affinity: {}
+  kyverno:
+    enabled: true
+    version: "2.5.1"
+    namespace: kyverno
+    nodeSelector: {}
+    tolerations: []
+    affinity: {}
+  kyverno_policies:
```

Comment:
Thinking either roll this up to `kyverno.policies`, or have `policies.kyverno`(and `policies.istio` later, etc).

Reply:
I'm not sure if  isito has a similar separate policy helm chart like this one but the kyverno policies is it's own chart with it's own values which could get a little messy if we did kyverno.policies and we want to expose any values here to override at this level., eg there's a value in the chart that selects the 'level' of security policies the chart provides eg baseline, enhanced, etc. and exposing it could end up looking like kyverno.policies.sublevel_thing . Unless we want to just patch all values for the policy chart?